utils/realize_patch.sh

```diff
@@ -1,14 +1,20 @@
 #!/bin/bash -eu
 
 set -o pipefail
-
-
 function main() {
     if [[ ${1:-} == '-v' ]]; then
         shift
         set -x
     fi
 
+    for template in $(find . -iname \*.tpl); do
+        realize_tempalte "$template"
```

Comment:
`s/tempalte/template/`

Reply:
Ahahahah, and the autocomplete made sure it was consistent across the file! Xd